fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -521,7 +521,7 @@ dropIndexStatement
     ;
 
 indexType
-    : USING (BITMAP | GIN)
+    : USING (BITMAP | GIN | NGRAMBF)
```

Comment:
add this word to nonReserved in StarRocks. If not , it may lead query select ngrambf from tbl failed if ngrambf is a col name in old version.

Reply:
good catch, fixed!